benchmark/Streamly/Benchmark/Data/ArrayOps.hs

```diff
@@ -37,45 +60,83 @@ value = 100000
 -- Stream generation and elimination
 -------------------------------------------------------------------------------
 
-type Stream = A.Array
+-------------------------------------------------------------------------------
+-- CPP Common to:
+-- MEMORY_ARRAY
+-- DATA_ARRAY
+-- DATA_PRIM_ARRAY
+-- DATA_SMALLARRAY
+-------------------------------------------------------------------------------
```

Comment:
These comments and other such comments can perhaps be removed, they don't seem to be useful.

Reply:
Once the module grows, CPP gets really messy. As long as these comments aren't negatively affecting the code, I am not in favor of removing them. They are comments for the developers anyway.